### DISCUSSION TITLE
Button Export not Shown

### DISCUSSION THREAD

**Discussion by zendzo** [pxlrbt/filament-excel#145] [Q&A]
![image](https://github.com/pxlrbt/filament-excel/assets/3133463/7613526f-52c3-4f7a-ad46-bf7735eef5e4)

this is happen on production server, on local server seems normally.

I tried to inspect every error but found nothing.

**Comment by pxlrbt** [comment]
Seems like there is no translation for your selected language and you removed `en` as `fallback_langauge`

**Comment by abewartech** [comment]
how to show the label?
